packages/lab/src/components/Section/Section.stories.tsx

```diff
@@ -0,0 +1,197 @@
+        </div>
+      </HvSection>
+    );
+  },
+};
+
+export const WithActions: StoryObj<HvSectionProps> = {
+  render: () => {
+    const classes = {
+      container: css({
+        display: "grid",
+        gridTemplateColumns: "repeat(2, 1fr)",
+        gap: theme.space.sm,
+      }),
+      root: css({ position: "relative" }),
+      content: css({
+        display: "flex",
+        flexDirection: "column",
+        justifyContent: "center",
+        alignItems: "center",
+        position: "absolute",
+        top: "50%",
+        left: "50%",
+        transform: "translate(-50%, -50%)",
+      }),
+    };
+
+    const data = {
+      Country: ["Portugal", "Spain", "France", "Germany"],
+      "Tickets Sold": [61829, 123948, 253792, 524638],
+    };
+
+    return (
+      <HvSection title="Simple section">
+        <div className={classes.container}>
+          <div>
+            <HvTypography>
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed
+              tempor blandit ipsum quis sollicitudin. Aliquam erat volutpat.
+              Praesent nisi nisl, sodales vitae blandit tincidunt, malesuada id
+              sapien. Nulla dapibus accumsan est, a pharetra velit consequat et.
+              Nullam iaculis justo sed urna condimentum ultricies. Integer nec
+              interdum tortor. Nulla molestie nibh in elit congue malesuada.
+              Donec fringilla volutpat sapien id maximus. Vestibulum faucibus
+              pellentesque ex, non gravida dui pharetra quis. Nulla facilisi.
+              Suspendisse erat nisl, mollis ut est nec, malesuada feugiat orci.
+              Vivamus dignissim nibh id lacinia vehicula.
+            </HvTypography>
+          </div>
+          <div className={classes.root}>
+            <HvDonutChart
+              data={data}
+              groupBy="Country"
+              measure="Tickets Sold"
+            />
+            <div className={classes.content}>
+              <Ticket iconSize="M" />
+              <HvTypography variant="title3">
+                {data["Tickets Sold"].reduce((acc, value) => acc + value, 0)}
+              </HvTypography>
+            </div>
+          </div>
+        </div>
+      </HvSection>
+    );
+  },
+};
+
+export const Multiple: StoryObj<HvSectionProps> = {
+  render: () => {
+    const [openIds, setOpenIds] = useState<string[]>([]);
+    const [multiple, setMultiple] = useState(false);
+    const sections = useMemo(
+      () => [
+        { id: "1", title: "Section 1", content: "Section 1 content." },
+        { id: "2", title: "Section 2", content: "Section 2 content." },
+        { id: "3", title: "Section 3", content: "Section 3 content." },
+        { id: "4", title: "Section 4", content: "Section 4 content." },
+        { id: "5", title: "Section 5", content: "Section 5 content." },
+        { id: "6", title: "Section 6", content: "Section 6 content." },
+      ],
+      []
+    );
+
+    useEffect(() => {
```

Comment:
Nothing major but I think we should remove this since since it's doing unnecessary work in the sample.